__tests__/app.test.js

```diff
@@ -83,8 +83,35 @@ describe("get /api/articles/:article_id", () => {
     });
 });
 
-
-
+describe("get /api/articles", () => {
+    test("return an array of article objects with the appropriate properties without a body property", () => {
+        return request(app)
+            .get("/api/articles")
+            .expect(200)
+            .then(({ body }) => {
+                expect(body.articles.length).toBeGreaterThan(0);
```

Comment:
Really good to have this length check here, but we could be more specific and expect a specific length as we know how many items are in our test database.